tests/test_validators.py

```diff
@@ -53,3 +57,26 @@ def test_validate(self):
         Validate that code runs without any errors.
         """
         self.program_metadata_validator.validate()
+
+
+class TestXBlockMetadataProviderValidator(TaxonomyTestCase):
+    """
+    Validate that validation logic does not have any errors.
+    """
+
+    def setUp(self):
+        """
+        Instantiate an instance of XBlockMetadataProviderValidator for use inside tests.
+        """
+        super(TestXBlockMetadataProviderValidator, self).setUp()
+        self.xblock = MockXBlock()
+
+        self.xblock_metadata_validator = XBlockMetadataProviderValidator(
+            [str(self.xblock.key)]
+        )
+
+    def test_validate(self):
+        """
+        Validate that code runs without any errors.
+        """
+        self.xblock_metadata_validator.validate()
```

Comment:
should we add a test case where we remove a key and validator validates the block to be wrong?

Reply:
It is just to make sure that the validator works as expected. These validators are used in [course-discovery](https://github.com/openedx/course-discovery/blob/master/course_discovery/apps/taxonomy_support/tests/test_providers.py) to test actual providers.